posts/Gunnar/2018-03-19-spec-api-modularity-patterns.adoc

```diff
@@ -0,0 +1,426 @@
+= "Spec API Modularity Patterns"
+Gunnar Morling
+:awestruct-tags: [ "Discussions" ]
+:awestruct-layout: blog-post
+---
+
+While Java 9 http://www.oracle.com/technetwork/java/eol-135779.html[reaches its end-of-life] this month,
+the Java Platform Module System (JPMS, https://www.jcp.org/en/jsr/detail?id=376[JSR 376]) is here to stay.
+That means that also specifications such as the https://www.jcp.org/en/jsr/detail?id=338[Java Persistence API] or https://www.jcp.org/en/jsr/detail?id=380[Bean Validation]
```

Comment:
> "That means that also specifications such as"

convoluted? Suggestion, reverse the "also" position

"That also means that specifications such as"

or shorter:

"That implies that specifications such as"